tests/Microsoft.CodeAnalysis.Testing/Directory.Build.props

```diff
@@ -10,6 +10,12 @@
   <Import Project="Sdk.props" Sdk="RoslynTools.RepoToolset" />
 
   <PropertyGroup>
+    <TestTargetFrameworks>net46;netcoreapp1.1;netcoreapp2.0</TestTargetFrameworks>
+    <!-- Uncomment the following line to test on .NET Core 1.1 -->
+    <!--<TestTargetFrameworks>netcoreapp1.1;$(TestTargetFrameworks)</TestTargetFrameworks>-->
+    <!-- Uncomment the following line to test on .NET Core 2.0 -->
+    <!--<TestTargetFrameworks>netcoreapp2.0;$(TestTargetFrameworks)</TestTargetFrameworks>-->
```

Comment:
delete these?

Reply:
Yes, after Test Explorer implements support for multi-targeted unit tests. 😄

Reply:
Where is the bug on test explorer?

Reply:
@Shyam-Gupta do you have a link to the bug for this? It's the multi-target situation we were discussing earlier today.